Snowflake.Data.Tests/SFBaseTest.cs

```diff
@@ -54,14 +55,14 @@ public class SFBaseTestAsync
         private Stopwatch _stopwatch;
         
         [SetUp]
-        public void BeforeTest()
+        public void SetupStopwatch()

```

Comment:
Why renaming this method? As I can see `BeforeTest` is the common name used across the project.

Reply:
I have renamed this method to make it more specific what is it's purpose. How about `SetupStopwatchBeforeTest`?

Reply:
I'm not sure. For me it's more like a standard and I would like to preserve the name consistent with other methods. See, this method is public, publicly it doesn't matter if internally some stopwatch is set or not. Publicly it is important to run it before each test.

Reply:
Renamed it.